acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/signer/Signer.java

```diff
@@ -38,6 +42,9 @@
   private final Optional<ClientTlsConfig> clientTlsConfig;
 
   public Signer(final SignerConfiguration signerConfig, final ClientTlsConfig clientTlsConfig) {
+    super(
+        new ObjectMapper().registerModule(new FilecoinJsonRpcModule()),
+        JSON_RPC_PATH + "/filecoin");
```

Comment:
nit: you can use `tech.pegasys.eth2signer.tests.AcceptanceTestBase#FC_RPC_PATH` instead.